#0-no, 1-yes
int8     isFromOutStart
string name
---
int32 ret